src/explore-education-statistics-common/src/components/__tests__/Accordion.test.tsx

```diff
@@ -112,7 +112,9 @@ describe('Accordion', () => {
     expect(contents[1]).toHaveAttribute('id', 'test-sections-2-content');
   });
 
-  test('scrolls to and opens section if location hash matches section heading ID', async () => {
+  // TODO: DFE-952
```

Comment:
I think the reason this is failing is because the `h2` on initial render doesn't actually have the heading ID assigned to it. See `AccordionSection`.